src/SDK/Language/PHP.php

```diff
@@ -10,8 +10,8 @@ class PHP extends Language {
      * @var array
      */
     protected $params = [
-        'composerVendor' => 'vendorName',
-        'composerPackage' => 'packageName',
+        'composerVendor' => 'vendor-name',
```

Comment:
Not sure why this change was made?

Reply:
Composer seem to have issue with uppercase letters in these values.